shared/elements/gaia_checkbox/style.css

```diff
@@ -13,6 +13,7 @@ gaia-checkbox {
 
 #checkbox span {
   font-size: 1.9rem;
+  font-weight: normal;
```

Comment:
This *is* upstream for 2.5. The components in shared/ are different than the components in the gaia-components repo.